SQLite/Core/Connection.swift

```diff
@@ -72,10 +273,32 @@ public final class Connection {
     ///     Default: `false`.
     ///
     /// - Returns: A new database connection.
-    public init(_ location: Location = .InMemory, readonly: Bool = false) throws {
+    public convenience init(_ location: Location = .InMemory, readonly: Bool = false, vfsName: String? = nil) throws {
```

Comment:
The documentation comment is now out-of-date. Also, based on the existing design of this class, I don't think it should now be leaking concepts like vfsName, flags, and dispatcher into the public API.